util/promote_staging_data

```diff
@@ -83,18 +83,14 @@ done
 
 # Try syncing staging data to production
 for team in "${TEAMS[@]}"; do
-  staging_bucket=gs://asap-staging-data-${team}
+  staging_bucket=gs://asap-dev-data-${team}
   production_bucket=gs://asap-curated-data-${team}
 
   log "Promoting [${team}] data to production"
   log "\tStaging bucket:\t\t[${staging_bucket}]"
   log "\tProduction bucket:\t[${production_bucket}]"
 
   gsync \
-    "${staging_bucket}/preprocess" \
-    "${production_bucket}/preprocess"
-
-  gsync \
-    "${staging_bucket}/cohort_analysis" \
-    "${production_bucket}/cohort_analysis"
```

Comment:
Had to remove these paths because `gsutil rsync -d` would only delete files in the destination bucket that are not in source bucket at these paths and not at top-level.